bls/.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 .idea
+algebra/target
```

Comment:
Maybe just ignore "target" so it ignores targets both in bls-zexe and bls-snark?

Reply:
Sure